src/app/ReadHandler.cpp

```diff
@@ -108,8 +128,33 @@ CHIP_ERROR ReadHandler::OnStatusReport(Messaging::ExchangeContext * apExchangeCo
                  err = CHIP_ERROR_INVALID_ARGUMENT);
     switch (mState)
     {
+    case HandlerState::SubscribingResponding:
+        ClearInitialReport();
```

Comment:
why is this a method? this seems the only usage and it is internal and all it does is 'mInitialReport = false;'

Could we simplify code?